__tests__/app.test.js

```diff
@@ -13,3 +23,19 @@ describe("GET /api", () => {
       });
   });
 });
+
+
+describe("GET /api/topics", () => {
+  test("200: Responds with an object containing the topics", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.topics.length).toBe(3);
```

Comment:
Well done for having a length check here to prevent a false pass if we get back an empty array!